src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -1,31 +1,61 @@
+<app-header></app-header>
+<div class="container-fluid screen-min-ht">
 <div *ngIf="workItem">
-  <h2>{{workItem.name}} details!</h2>
-  <div>
-    <label>id: </label>{{workItem.id}}</div>
-  <div>
-    <label>fields owner: </label>
-    <input [(ngModel)]="workItem.fields['system.owner']" placeholder="owner" />
+  <h1>{{workItem.name}} Details</h1>
+  <br />
+  <div class="form-horizontal">
+    <div class="form-group">
+      <label class="col-sm-1 control-label">ID</label>
+      <div class="col-sm-5">
+        {{workItem.id}}
+      </div>
+    </div>
+    <div class="form-group">
+      <label class="col-sm-1 control-label" for="name">Name</label>
```

Comment:
The use of 'for' does not have anything to refer to. You need to add an 'id' or 'name' attribute to the div the label is 'for'.